docs/static/schema.json

```diff
@@ -348,17 +348,9 @@
       "type": "object",
       "properties": {
         "version": {
-          "description": "Specify the Taskfile format that this file conforms to.",
-          "anyOf": [
-            {
-              "type": "number",
-              "enum": [3]
-            },
-            {
-              "type": "string",
-              "enum": ["3"]
-            }
-          ]
+          "description": "A version for the current Taskfile",
+          "type": "string",
+          "enum": ["2", "3"]
```

Comment:
Can you please keep the support for `number` as before?

Reply:
Is there something wrong with my change? Yes, I can keep as before, but it doesn't explain why.

Reply:
`version: 3` is valid. Quotes like in `version: '3'` are not required.